lib/core.js

```diff
@@ -226,6 +226,7 @@ declare class $ReadOnlyArray<+T> {
     keys(): Iterator<number>;
     lastIndexOf(searchElement: mixed, fromIndex?: number): number;
     map<U>(callbackfn: (value: T, index: number, array: $ReadOnlyArray<T>) => U, thisArg?: any): Array<U>;
```

Comment:
why does this return `$ReadOnlyArray<U>` while map returns just `U`?

Reply:
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap